ddtrace/tracer/tracer_test.go

```diff
@@ -322,6 +332,21 @@ func TestStartSpanOrigin(t *testing.T) {
 	assert.Equal("synthetics", carrier2[originHeader])
 }
 
+func TestMeasuredSpan(t *testing.T) {
+	// Create three spans for a single trace.
+	// The first two are measured but the third one is not.
+	assert := assert.New(t)
+
+	tracer := newTracer(WithServiceName("my-service"))
+	root := tracer.StartSpan("web.request", ResourceName("/resource"), MeasureSpan()).(*span)
+	assert.Equal(float64(1), root.Metrics[keyMeasured])
+	child := tracer.StartSpan("custom_operation", ChildOf(root.Context()), MeasureSpan()).(*span)
+	assert.Equal(float64(1), child.Metrics[keyMeasured])
+	// child2 will not inherit the measured flag
```

Comment:
Why would it? What's the point of this entire test? Is there one? If not, can we remove it?